postman/Move Subscriber.postman_collection.json

```diff
@@ -0,0 +1,355 @@
+							" ",
+							"pm.test(\"Successfully get a  random list\", function() {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"At least two lists found\", function() {",
+							"",
+							"    pm.expect(jsonData.total_size).to.be.above(1);",
+							"    pm.globals.set(\"source_listId\",jsonData.entries[0].id );",
+							"    pm.globals.set(\"destination_list_self_link\",jsonData.entries[1].self_link );",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Subscribers on Source List",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "75df1ad8-7a66-40eb-b8e7-2991fcd51619",
+						"exec": [
+							"pm.test(\"Successfully get a list\", function(){",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"List has at least one subscriber\", function() {",
+							"   var jsonData = pm.response.json() ;",
+							"   pm.expect(jsonData.total_size).to.be.above(0);",
+							"   pm.globals.set(\"subscriber_link\", jsonData.entries[0].self_link);",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{source_listId}}/subscribers",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{source_listId}}",
+						"subscribers"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Subscriber",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "8e652d7e-fd45-4591-a567-b0c9589d9ce6",
+						"exec": [
+							"pm.test(\"Successfully requested subscriber info\", function(){",
+							"    pm.response.to.have.status(200);",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{subscriber_link}}",
+					"host": [
+						"{{subscriber_link}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Move subscriber",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "054015e7-1062-4de8-9dc2-8e4b84849b25",
+						"exec": [
+							" pm.globals.set(\"new_subscriber_url\", '');",
+							" ",
+							" pm.test(\"Successfully moved subscriber\", function(){",
+							"    if (pm.response.to.have.status(201)) {",
```

Comment:
This looks like an assertion, I think you can just run it without checking the return value.